model/v2/jsonid.go

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) 2023 The Jaeger Authors.
+// Copyright The OpenTelemetry Authors
```

Comment:
Copied from OTEL, but had to change to marshal IDs as base64, to keep with the original grpc-gateway version.